src/components/Calls/CallTime.js

```diff
@@ -0,0 +1,31 @@
+import format from 'date-fns/format';
+import { parseISO } from 'date-fns/fp';
+import isToday from 'date-fns/isToday';
+import { h } from 'preact';
+
+import I18n from '../../i18n';
+import { createClassName } from '../helpers';
+import styles from './styles.scss';
+
+
+export const CallTime = (props) => {
+	const hh = parseInt(Math.abs(props.endTime - props.time) / 36e5);
```

Comment:
In which format are we receivig endTime and time? Wouldn't be possible to use a library to do the conversion to "human time" instead of doing it manually?

Reply:
Both these variables contain an ISO string. Regarding using the library for calculating duration, I think that would be a good idea. So we're now using [this](https://date-fns.org/v2.25.0/docs/formatDistance) method to do this calculation. No additional import was required since we're already using `date-fns` package in this repo